applications/StructuralMechanicsApplication/custom_elements/truss_elements/linear_truss_element_2D.cpp

```diff
@@ -644,6 +644,54 @@ void LinearTrussElement2D<TNNodes>::CalculateOnIntegrationPoints(
 /***********************************************************************************/
 /***********************************************************************************/
 
+template<SizeType TNNodes>
+void LinearTrussElement2D<TNNodes>::CalculateOnIntegrationPoints(
+    const Variable<Vector>& rVariable,
+    std::vector<Vector>& rOutput,
+    const ProcessInfo& rProcessInfo
+    )
+{
+    const auto& integration_points = IntegrationPoints(GetIntegrationMethod());
+    rOutput.resize(integration_points.size());
+
+    if (rVariable == PK2_STRESS_VECTOR) {
+        ConstitutiveLaw::Parameters cl_values(GetGeometry(), GetProperties(), rProcessInfo);
+        cl_values.GetOptions().Set(ConstitutiveLaw::COMPUTE_STRESS             , true);
+        cl_values.GetOptions().Set(ConstitutiveLaw::COMPUTE_CONSTITUTIVE_TENSOR, true);
+
+        const double length = CalculateLength();
+
+        // Let's initialize the cl values
+        VectorType strain_vector(1), stress_vector(1);
+        MatrixType C(1,1);
+        strain_vector.clear();
+        cl_values.SetStrainVector(strain_vector);
+        cl_values.SetStressVector(stress_vector);
+        cl_values.SetConstitutiveMatrix(C);
+        SystemSizeBoundedArrayType nodal_values(SystemSize);
+        GetNodalValuesVector(nodal_values);
+
+        SystemSizeBoundedArrayType B;
+
+        // Loop over the integration points
+        for (SizeType IP = 0; IP < integration_points.size(); ++IP) {
```

Comment:
The Kratos style guide ( https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide ) says lower case with underscores for local variables. Here that would result in:, c, b, integration_point i.s.o. C, B, IP

Reply:
Thank you for the comment. Fixed names and replaced C/B with longer names.